be/src/storage/rowset/column_iterator.h

```diff
@@ -259,9 +261,11 @@ class ColumnIterator {
 
     virtual Status null_count(size_t* count) { return Status::OK(); };
 
+    // RAW interface, should be used carefully
+    virtual ColumnReader* get_column_reader() { return nullptr; };
```

Comment:
suggest to 
```
virtual ColumnReader* get_column_reader()=0;
```
or use std::optional<ColumnReader*> as a return type to tell sb that the return type must be checked

Reply:
add an check in it, actually the unsupported `ColumnIterator` should not be called.